src/components/WorkerTable/index.jsx

```diff
@@ -0,0 +1,173 @@
+import { Component } from 'react';
+import { Link } from 'react-router-dom';
+import { TableCell, TableRow } from 'material-ui/Table';
+import { camelCase } from 'change-case';
+import { memoizeWith } from 'ramda';
+import LinkIcon from 'mdi-react/LinkIcon';
+import { withStyles } from 'material-ui/styles';
+import { ListItemText } from 'material-ui/List';
+import Typography from 'material-ui/Typography';
+import ContentCopyIcon from 'mdi-react/ContentCopyIcon';
+import InformationVariantIcon from 'mdi-react/InformationVariantIcon';
+import ButtonDrawer from '../ButtonDrawer';
+import TableCellListItem from '../TableCellListItem';
+import DateDistance from '../DateDistance';
+import DataTable from '../DataTable';
+import TaskMetadataCard from '../TaskMetadataCard';
+import StatusLabel from '../StatusLabel';
+import { worker } from '../../utils/prop-types';
+import sort from '../../utils/sort';
+
+@withStyles(theme => ({
+  dateListItem: {
+    marginLeft: -theme.spacing.unit,
+    padding: theme.spacing.unit,
+  },
+  listItemButton: {
+    ...theme.mixins.listItemButton,
+  },
+  taskName: {
```

Comment:
Can these styles by applied to the cell instead of creating an inner div and applying them?

Reply:
@eliperelman There's two elements inside `TableCellListItem`. It would be hard to target the text node without wrapping it in a `div`. I'm open to suggestions.